server/@types/hmpo-form-wizard/index.d.ts

```diff
@@ -25,20 +25,34 @@ declare module 'hmpo-form-wizard' {
     /** Use in complex generic forms that might allow multiple values in any fields; have checkbox components */
     type MultiValues = Record<string, MultiValue>
 
+    /** Operator function, returns true if submitted values matches condition */
+    type OperatorFunction = (
+      submittedValue: Value | MultiValue,
+      req: Request,
+      res: Express.Response,
+      condition: unknown,
```

Comment:
(for future reference: i _think_ condition is `NextStep` or `NextStepCondition` but haven't confirmed it)